pkg/agent/proxy/endpoints.go

```diff
@@ -95,10 +108,42 @@ func (t *endpointsChangesTracker) OnEndpointUpdate(previous, current *corev1.End
 	return len(t.changes) > 0
 }
 
+// EndpointSliceUpdate updates given the service's endpoints change map based on the <previous, current> endpoints pair.
```

Comment:
same as above: "updates the given Service's endpointsChange map based on the"

Reply:
Done